qemu/tests/cfg/block_stream.cfg

```diff
@@ -108,3 +124,37 @@
                     when_streaming = "pause_job set_speed resume_job cancel start wait_for_finished"
                     Host_RHEL.m6:
                         when_streaming = "set_speed cancel start wait_for_finished"
+        - data_integrity:
+            type = block_stream_simple
+            before_start = "create_files stop"
+            after_finished = "resume verify_md5s"
+        - stop_cont:
+           type = block_stream_simple
+           before_start = "create_files"
+           when_streaming = "stop resume stop resume stop resume"
+           after_finished = "reboot verify_alive"
+        - with_installation:
+            type = block_stream_installation
+            wait_timeout = 36000
+            default_speed = 10240000
+            need_install = yes
+            start_vm = no
+            images = "stg"
+            image_name_stg = "images/block_stream_installation"
+            image_size_stg = 20G
+            boot_drive_stg = yes
+            source_image = "stg"
```

Comment:
In this case, `source_image`  and  `images` are same,  as system disk.  No other disk be added to 'images' during this test.  I think only `images` is enough.   But 'source_image' is more  understandable from another point of view.  It is up to you.

Reply:
@yanan-fu Yes, that's a concept shared with other test cases, it will be more clear when there are multiple blocks. I would like to keep it. Thanks.